src/Microsoft.DotNet.Web.Client.ItemTemplates/Microsoft.DotNet.Web.Client.ItemTemplates.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
-    <Description>Web Client-Side File Templates for Microsoft Template Engine</Description>
+    <Description>Web Client-Side File Templates for Microsoft Template Engine. Can only be installed with `dotnet new --install`.</Description>
```

Comment:
I think it's odd to use Markdown back-ticks in a plain-text description. Maybe just use regular apostrophes?

Reply:
Nit: I would add a new line and make it a full sentence.

```xml
<Description>Web Client-Side File Templates for Microsoft Template Engine

To install the templates in this package, run 'dotnet new --install $(PackageId)::$(PackageVersion)'.
</Description>
```

Reply:
I would be OK with that, depending on how it's actually displayed to the user.

Reply:
It should appear like this:


![image](https://user-images.githubusercontent.com/2696087/46560514-e5c8c080-c8a8-11e8-8285-92bcafb885d8.png)

Reply:
That looks nice.